### DISCUSSION TITLE
RSContext layer decisions

### DISCUSSION THREAD

**Discussion by jtgilbert** [Riverscapes/riverscapes-tools#616] [General] [answered]
As I work on final updates I just figured it would be good to get some input on a few things. In general, I am thinking RSContext should house datasets that are either useful as context (in a watershed assessment type sense) or are inputs to downstream tools. I am open to discussion, but with that in mind:
- I removed flow accumulation and drainage area, as they are performed in TauDEM. I have not removed Slope, but probably should following the same logic
- The fair market value raster may or may not be useful context...but we don't use it for any downstream tools, and it hasn't been updated for a while despite the changes in real estate since COVID. I wonder if it is too static of a dataset to be super useful?
- The new surface water rasters could be useful context, but I can also see logic for them being their own project.

thoughts @joewheaton @philipbaileynar @MattReimer ?

**Comment by philipbaileynar** [comment, marked as answer]
> datasets that are either useful as context (in a watershed assessment type sense) or are inputs to downstream tools

- I really like this statement but I think that we should consider it a "guideline" rather than a rule. I like the decisions you have made and say move on! 
- Slope is such an elementary dataset that I say that it should remain in RSC.
- I don't have a strong opinion above FMV and surface water. I would just say that the "one-stop-shop" aspect of RSC is valuable. It kind of defeats the object if users have to go download several projects... They might as well go to the original sources at that point!

**Reply by MattReimer** [reply]
Yeah. Same. That FMV raster was useful on its own in tools like WebRAVE but I'm not sure what state the metadata is in or if we need to store multiple versions of it per year etc.

**Reply by joewheaton** [reply]
- I concur with @jtgilbert, @philipbaileynar and @MattReimer on removal of flow accumulation and drainage area, as they are performed in TauDEM.
- I don't really mind either way on Slope... happy to leave it as @philipbaileynar suggests.
- Lets keep FMV. It will get used (and potentially updated later).  It is only one layer. We can find updates later. 
- Lets move the surface water stuff out into its own thing. It is a good exercise and will shine a spot light on a stand alone product produced by others and make it more useful. Since there is already a multi-temporal synthesis @MattReimer (it is based on entire record of LANDSAT), it can be updated periodically by the authors. As such, having in its own project will make it easier to release as "versions". This is different then TauDEM. We are shining a spotlight on TauDEM by running an existing tool and packaging it up nicely and displaying it. By contrast, with this, we are just gobbling up something that has been run by others and making it available as a riverscapes project.

**Reply by joewheaton** [reply]
> * I don't have a strong opinion above FMV and surface water. I would just say that the "one-stop-shop" aspect of RSC is valuable. It kind of defeats the object if users have to go download several projects... They might as well go to the original sources at that point!

I strongly disagree with this point that "they might as well go to the original source".  The original source IS NOT as convenient as what we offer to GIS users that are Riverscapes Consortium users.
